MIDIchlorians/MIDIchloriansTests/AnimationBitTests.swift

```diff
@@ -41,4 +41,5 @@ class AnimationBitTests: XCTestCase {
         )
         XCTAssertNil(animationBit)
     }
+
```

Comment:
oh, I think I commented it out earlier and didn't bother to remember if it had newline or not haha